regtests/mww3_test_09/input/subGtLksgrid

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
```

Comment:
Should this stay in the main repo?  Also I couldn't comment on the .dat files themselves, but should these be in the repo or in the ftp bundle? GitHub wouldn't load many of the them so I wasn't sure what needed to be done there?

Reply:
These *.dat files are input files for the sample grid and they are better kept with the example programs.  They do not need any update so far.

Reply:
I think this particular file should not be in the commit (it is a script to run the test on our HPC). I think you can remove this one @ukmo-jianguo-li .

As for the *.dat files, I would agree with @JessicaMeixner-NOAA - these should probably go in the ftp bundle. @ukmo-jianguo-li : when there are large input files associated with a regression test, the convention has been to put these in a separate "regtest data" tarball that is downloaded separately.